data/adapters/encode_E2G_CRISPR_adapter.py

```diff
@@ -0,0 +1,100 @@
+                chr, start, end = region_coordinate.split(',')
+                _id = build_regulatory_region_id(chr, start, end, 'CRISPR')
+
+                _props = {
+                    'name': _id,
+                    'chr': chr,
+                    'start': start,
+                    'end': end,
+                    'type': region_type,
+                    'source': ENCODE2GCRISPR.SOURCE,
+                    'source_url': ENCODE2GCRISPR.SOURCE_URL
+                }
+
+                yield(_id, self.label, _props)
+
+        elif self.label == 'regulatory_region_gene':
+            with open(self.filepath, 'r') as crispr_file:
+                crispr_csv = csv.reader(crispr_file, delimiter='\t')
+                next(crispr_csv)
+                for row in crispr_csv:
+                    gene_id = row[14]
+                    if gene_id == 'NA':
+                        continue
+
+                    chr = row[0]
+                    start = row[1]
+                    end = row[2]
+                    score = row[4]  # effect size, need to take care of NA rows
+                    p_value = row[19]  # pValueAdjusted
+
+                    regulatory_region_id = build_regulatory_region_id(
+                        chr, start, end, 'CRISPR')
+
+                    _id = regulatory_region_id + '_' + gene_id + '_' + ENCODE2GCRISPR.FILE_ACCESSION
+                    _source = 'regulatory_regions/' + regulatory_region_id
+                    _target = 'genes/' + gene_id
+                    _props = {
+                        'score': score,
+                        'p_value': p_value,
+                        'source': ENCODE2GCRISPR.SOURCE,
+                        'source_url': ENCODE2GCRISPR.SOURCE_URL,
+                        'biological_context': 'ontology_terms/' + ENCODE2GCRISPR.BIOLOGICAL_CONTEXT
+                    }
+                    yield(_id, _source, _target, self.label, _props)
+
+    def load_regulatory_region(self):
+        # each row is a pair of tested regulatory region <-> gene, significant column can be TRUE/FALSE
+        # one regulatory region can be tested in multiple rows, i.e. with multiple genes
+        # we want to assign type = 'enhancer' if the regulatory region has significant = 'TRUE' with any tested gene, else assign type = 'CRISPR_tested_element'
+        # store those info in a dictionary here and output all nodes info at the end, since the file is not big (3,962 unique regions tested)
+        self.regulatory_region_nodes = {}
+
+        with open(self.filepath, 'r') as crispr_file:
+            crispr_csv = csv.reader(crispr_file, delimiter='\t')
+            next(crispr_csv)
+            for row in crispr_csv:
+                regulatory_region_coordinate = ','.join(row[:3])
+
+                significant = row[17]
+
+                if self.regulatory_region_nodes.get(regulatory_region_coordinate) is None:
```

Comment:
this is good! usually people write this conditional using `in`, something like:
`if regulatory_region_coordinate not in self.regulatory_region_nodes:`

Reply:
got it thanks!